### ISSUE TITLE
Theming: aspect ratio of logo

### ISSUE THREAD

**Issue by jospoortvliet** [nextcloud/server#344]
I added a round image as theme and whadda-you-know - it didn't look so awesome:
![spectacle p16387](https://cloud.githubusercontent.com/assets/551757/16685381/1aee5898-450b-11e6-9705-3ca6f910e984.png)

I think we should either preserve the aspect ratio (best) or tell users what aspect ratio the logo must have ;-)

Shouldn't be very hard to fix.

**Comment by mar1u5** [comment]
- [ ] add some information about optimal size (px) for uploaded pictures

**Comment by juliushaertl** [comment]
There is also an issue on the login page. Images that are too high, get cropped there.

**Comment by jospoortvliet** [comment]
@juliushaertl ah yes they do...
![spectacle f29735](https://cloud.githubusercontent.com/assets/551757/16703005/4b8c90b4-456a-11e6-830b-eff6034760da.png)

**Comment by Bugsbane** [comment]
Personally I'd recommend keeping the aspect ration with max-height and max-width values on both images.

**Comment by clorenz150** [comment]
In apps/theming/lib/controller/themingcontroller.php at line 223 I replaced the content of `#header .logo-icon` with:
```css
width: 62px;
height: 34px;
background-image: url(\'./logo?v='.$cacheBusterValue.'\');
background-size: contain;
vertical-align:middle;
display: inline-block;
```

Looks fine for me, but I'm not sure if this has site affects.

**Comment by Bugsbane** [comment]
Looks good @clorenz150 👍

**Comment by MorrisJobke** [comment]
@clorenz150 Could I ask you to open this change as a pull request?

**Comment by christianpoortvliet** [comment]
Does not seem to work with transparant png's too, so that is a bummer.

**Comment by christianpoortvliet** [comment]
getting back to default color for theming not also

**Comment by Bugsbane** [comment]
> Does not seem to work with transparant png's

I imagine this is likely a browser specific issue, although I'd need to check. What browser are you viewing on?

**Comment by Bugsbane** [comment]
Sorry, hit the wrong button. Didn't mean to close this issue.

**Comment by MorrisJobke** [comment]
Fixed with  #525
